<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferWindow</class>
 <widget class="QWidget" name="TransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>775</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transfer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/rec/pictures/free-icon-atm-5346812.png</normaloff>:/rec/pictures/free-icon-atm-5346812.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-5</y>
     <width>800</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:#006400;</string>
   </property>
   <property name="text">
    <string>Перевод стредств</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="sum_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>771</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>731</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="insertButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.921631, y1:1, x2:0.917, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #006400;
font-size: 17px;
font-weight: 700;
margin-left:10px;
margin-right: 15px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Ввод</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.122749, y1:1, x2:0.129, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #800000;
font-size: 17px;
font-weight: 700;
margin-left: 30px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="correctButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.921631, y1:1, x2:0.917, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #FFD700;
font-size: 17px;
font-weight: 700;
margin-left:10px;
margin-right: 15px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Введите сумму перевода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sumLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>361</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QLabel" name="minSumLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;</string>
    </property>
    <property name="text">
     <string>Минимальная сумма: 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cardNumber_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="cardNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="placeholderText">
     <string>Номер карты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>761</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Введите номер карточки 
 получателя перевода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>741</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="insertCardButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.921631, y1:1, x2:0.917, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #006400;
font-size: 17px;
font-weight: 700;
margin-left:10px;
margin-right: 15px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Ввод</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="cancelCardButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.122749, y1:1, x2:0.129, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #800000;
font-size: 17px;
font-weight: 700;
margin-left: 30px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="corrcetCardButton">
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>41</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:qlineargradient(spread:pad, x1:0.921631, y1:1, x2:0.917, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #FFD700;
font-size: 17px;
font-weight: 700;
margin-left:10px;
margin-right: 15px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="wait_group">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>-30</y>
     <width>431</width>
     <height>491</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>454</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подождите...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="noMoney_forSend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>741</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>Перевод нельзя осуществить, проверьте баланс на карте
или же введите корректную сумму</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="enterCorectLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>Введите корректный номер карты!</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
